9.2/run-postgresql.sh

```diff
@@ -90,12 +107,16 @@ function initialize_database() {
 
 		# Allow connections from all hosts.
 		host all all all md5
+
+		# Allow replication connections from all hosts.
+		host replication all all md5
 	EOF
 
 	pg_ctl -w start
 	createuser "$POSTGRESQL_USER"
 	createdb --owner="$POSTGRESQL_USER" "$POSTGRESQL_DATABASE"
 	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH ENCRYPTED PASSWORD '${POSTGRESQL_PASSWORD}';"
+	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH REPLICATION ;"
```

Comment:
That's the thing, POSTGRESQL_USER is supposed to be an ordinary user without privileges. But from what you're saying, I guess the best approach would be to use the `postgres` account and give it the `REPLICATION` role, unless it already has it. We'd then require the admin password for replication to work.

@jmccormick2001 I don't want to waste your time with this. I'll be happy to rework this logic later, but I would like there to be a test for replication enablement, so that we don't have any impact on non-replication use cases.